pages/level.jsx

```diff
@@ -0,0 +1,31 @@
+import Button from "../components/button/button";
+import MainContainer from "../container/mainContainer/mainContainer";
+import Link from "next/dist/client/link";
+
+const BTN = [
+  { title: "NORMAL MODE", type: "mode" },
+  { title: "SUPER DUPER", type: "mode" },
+];
+const Level = () => {
+  return (
+    <MainContainer>
+      <section className="flex flex-col items-center ">
+        <h1 className=" text-4xl font-bold py-2">레벨을 골라주세요</h1>
+        <p>각 레벨에 대한 설명을 보여주고 이동하도록</p>
+      </section>
+      <section className="flex justify-around items-center w-80">
+        {BTN.map((items) => {
+          const { title, type } = items;
```

Comment:
```suggestion
        {BTN.map(({ title, type }) => {
```

이렇게 단축해서 코드를 쓸수도 있어요

Reply:
헐 이렇게도 되네요 진짜 이건 ㄹㅇ 센세이션🥹🥹🥹🥹🥹🥹